lib/wallet/txdb.js

```diff
@@ -2240,6 +2240,19 @@ class TXDB {
     });
   }
 
+  /**
+   * Test whether an account owns a transaction.
+   * @param {Number} acct
+   * @param {Hash} hash
+   * @returns {Promise} - Returns Boolean.
+   */
+
+  acctOwnsTX(acct, hash) {
```

Comment:
I don't love this method name. It feels out of style.

Reply:
Maybe `acctHasTX()`?